models/Recipe.js

```diff
@@ -1,9 +1,17 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
-  // TODO: write the schema
+  title: { type: String },
```

Comment:
Se só tem um parâmetro, não precisa das chaves.
```suggestion
  title: String,
```